CommonCompilerConfig.cmake

```diff
@@ -91,6 +91,8 @@ set(EXTERNAL_CXX_FLAGS_OPTIMIZED)
 set(EXTERNAL_LINKER_FLAGS_UNOPTIMIZED)
 set(EXTERNAL_LINKER_FLAGS_OPTIMIZED)
 
+set(COMMON_STATICLIB_CMAKE_DEFINITIONS_OPTIMIZED)
+set(COMMON_STATICLIB_CONFIGURE_DEFINITIONS_OPTIMIZED_STR "")
```

Comment:
The file header contains documentation of each variable exported by this module. Pls document the two new variables there, too.